src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/KeyValueWatcher.cs

```diff
@@ -33,6 +33,11 @@ internal class KeyValueWatcher
         /// </summary>
         public DateTimeOffset CacheExpires { get; set; }
 
+        /// <summary>
+        /// The number of times we tried to reload this key-value with exponential backoff strategy.
+        /// </summary>
+        public int RefreshAttempts { get; set; } = 0;
```

Comment:
Why default it to 0? Also, we don't do that in CachedKeyVaultSecret

Reply:
We default it to 0 in the CachedKeyVaultSecret constructor.